lib/lita/handlers/irkit.rb

```diff
@@ -4,33 +4,32 @@
 module Lita
   module Handlers
     class Irkit < Handler
-      def self.default_config(handler_config)
-        handler_config.deviceid  = ENV['IRKIT_DEVICEID']
-        handler_config.clientkey = ENV['IRKIT_CLIENTKEY']
-      end
+      config :deviceid,  required: true, default: ENV['IRKIT_DEVICEID']
+      config :clientkey, required: true, default: ENV['IRKIT_CLIENTKEY']
 
       route /^ir list/,            :ir_list,         command: false, help: { "ir list"                      => "list irkit command names" }
       route /^ir send (.+)/,       :ir_send,         command: false, help: { "ir send [command_name]"       => "send irkit command" }
       route /^ir all off/,         :ir_send_all_off, command: false, help: { "ir all off"                   => "send irkit commands which end with 'off'" }
       route /^ir register (.+)/,   :ir_register,     command: true,  help: { "ir register [command_name]"   => "register irkit command" }
       route /^ir unregister (.+)/, :ir_unregister,   command: true,  help: { "ir unregister [command_name]" => "unregister irkit command" }
+      route /^ir migrate/,         :ir_migrate,      command: true
```

Comment:
TODO for me: Add `ir migrate` to https://github.com/fukayatsu/lita-irkit#usage and describe migrating